Source/SimpleActions/Public/Structs/SimpleActionList.h

```diff
@@ -20,8 +20,13 @@ struct SIMPLEACTIONS_API FSimpleActionList
 	
 public:
 
-	UPROPERTY(EditAnywhere, BlueprintReadOnly, Instanced, meta = (ShowOnlyInnerProperties))
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Instanced)
 	TArray<TObjectPtr<USimpleAction>> Actions;
 
+	FSimpleActionList() : FSimpleActionList(TArray<USimpleAction*>()) {}
```

Comment:
Does this require `TArray<USimpleAction*>()`? shouldn't it be initialized without that?

Reply:
I thought that too, but it complained when I took it out.